src/main/java/ua/lviv/iot/parkingServer/service/ParkingService.kt

```diff
@@ -1,51 +1,28 @@
 package ua.lviv.iot.parkingServer.service
 
-import jakarta.annotation.PostConstruct
-import jakarta.annotation.PreDestroy
-import java.util.concurrent.ConcurrentHashMap
-import java.util.concurrent.atomic.AtomicLong
 import org.springframework.stereotype.Service
 import ua.lviv.iot.parkingServer.annotation.TrackMetrics
-import ua.lviv.iot.parkingServer.datastorage.ParkingFileStore
+import ua.lviv.iot.parkingServer.repository.ParkingMongoRepository
 import ua.lviv.iot.parkingServer.exception.EntityNotFoundException
 import ua.lviv.iot.parkingServer.model.Parking
 import ua.lviv.iot.parkingServer.service.interfaces.ParkingServiceInterface
 
 @TrackMetrics
 @Service
-class ParkingService(private val parkingFileStore: ParkingFileStore) : ParkingServiceInterface {
-    private val parking: MutableMap<Long, Parking> = ConcurrentHashMap()
-    private var id: AtomicLong = AtomicLong(1L)
+class ParkingService(private val parkingMongoRepository: ParkingMongoRepository) : ParkingServiceInterface {
+    override fun findAllEntities(): List<Parking> = parkingMongoRepository.findAll()
 
-    override fun findAllEntities(): List<Parking> = parking.values.toList()
+    override fun findEntityById(id: Long): Parking = parkingMongoRepository.findById(id)
```

Comment:
why id is Long
in your entity class (Parking it is String)?

Reply:
Please fix all the places

Reply:
✅